samcli/cli/cli_config_file.py

```diff
@@ -263,4 +264,4 @@ def decorator_customize_config_env(f):
     return click.option(*config_env_param_decls, **config_env_attrs)(f)
 
 
-# End section copied from [[click_config_file][https://github.com/phha/click_config_file/blob/master/click_config_file.py]
+# End section copied from [click_config_file][https://github.com/phha/click_config_file/blob/master/click_config_file.py
```

Comment:
[minor] will the hyperlink work with the last close bracket missed?

Reply:
it is a python file so hyperlink won't work anyway

Reply:
I don't feel the original comment was trying to follow format like markdown 🤔